liblouisutdml/transcribe_paragraph.c

```diff
@@ -343,12 +343,6 @@ transcribe_paragraph (xmlNode * node, int action)
     }
   else if ((style = is_style (node)) != NULL)
     {
-      if (node->children == NULL)
```

Comment:
Have we gone through this? Why is this gone?

Reply:
See individual commit messages.